test/test_util.py

```diff
@@ -41,7 +41,7 @@ def test_hash_params_sha1_base32(self):
                               ('test/OmicsIntegrator1/output', PurePosixPath('/spras'), '/spras/TNDO5TR/output'),
                               ('../src', '/spras', '/spras/NNBVZ6X/src')])
     def test_prepare_volume(self, filename, volume_base, expected_filename):
-        _, container_filename = prepare_volume(filename, volume_base)
+        _, container_filename = prepare_volume(filename, volume_base, config.config.container_settings)
```

Comment:
This gets us a default `ProcessedContainerSettings`?

Reply:
(We should provide that explicitly instead of misdirecting it with the `container_settings` from the global config :+1:)